src/test/java/testPages/HomePageTest.java

```diff
@@ -0,0 +1,23 @@
+package testPages;
+
+import base.TestBase;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import steps.HomePageTestSteps;
+
+public class HomePageTest extends TestBase {
+    HomePageTestSteps homePageSteps;
+
+    @BeforeClass
+    public void setUp() {
```

Comment:
setUp is smth what relates to all the tests, right? would it make sense to store it in BaseTest class?